app/core/enterprise_auth.py

```
@@ -0,0 +1,632 @@
+                "user_agent": user_agent,
+                "access_token_hash": hashlib.sha256(access_token.encode()).hexdigest(),
+                "refresh_token_hash": hashlib.sha256(refresh_token.encode()).hexdigest(),
+                "expires_at": datetime.now(timezone.utc) + timedelta(days=self.refresh_token_expire_days),
+                "is_active": True,
+                "created_at": datetime.now(timezone.utc)
+            }
+            
+            # Create session record (assuming UserSession model exists)
+            try:
+                await enterprise_db.create_record(UserSession, session_data, session)
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)